lib/node_modules/@stdlib/stats/base/nanmskmax/docs/types/test.ts

```diff
@@ -26,7 +27,7 @@ import nanmskmax = require( './index' );
 	const x = new Float64Array( 10 );
 	const mask = new Uint8Array( 10 );
 
-	nanmskmax( x.length, x, 1, mask, 1 ); // $ExpectType number
+	nanmskmax( x.length, new AccessorArray( x ), 1, mask, 1 ); // $ExpectType number
```

Comment:
```suggestion
	nanmskmax( x.length, x, 1, mask, 1 );
	nanmskmax( x.length, new AccessorArray( x ), 1, mask, 1 ); // $ExpectType number
```
don't remove the old test, add a new one for accessor arrays

Reply:
same comment for the `ndarray` API below